vita3k/regmgr/src/regmgr.cpp

```diff
@@ -271,49 +268,57 @@ static void init_system_dreg(RegMgrState &regmgr, const std::wstring pref_path)
 
     for (const auto &reg : reg_template) {
         for (const auto &entry : reg.second) {
-            regmgr.system_dreg[reg.first][entry.name] = entry.value;
+            const std::vector<char> default_value(entry.value.begin(), entry.value.end());
+            auto &value = regmgr.system_dreg[reg.first][entry.name];
+            value.resize(entry.size);
+            value = default_value;
```

Comment:
```suggestion
            auto &value = regmgr.system_dreg[reg.first][entry.name];
            value.assign(entry.value.begin(), entry.value.end());
```

Reply:
forget this idea :) i have try is shit and get crazy shit inside file
only works fine like this

Reply:
and ho, it very not like too if you set value inside vector without resize
resize is request before assign too

Reply:
for more explain
that is when i using this code
![image](https://github.com/Vita3K/Vita3K/assets/5261759/7fc04bd4-d068-4fb1-a9de-fac0fc78e8be)
i have no used it for this reason

Reply:
ok is done, sorry i have bad read, but just important, all is empty here, so is obligatory to resize for using assign
if not get big crash